src/js/core/widget/core/Popup.js

```diff
@@ -644,6 +648,23 @@
 				eventUtils.on(self._ui.page, "pagebeforehide", self, false);
 				eventUtils.on(window, "resize", self, false);
 				eventUtils.on(document, "vclick", self, false);
+
+				self._ui.content.addEventListener("scroll", function () {
+					eventUtils.trigger(self._ui.content, "scrollstart");
```

Comment:
property `self._ui.content` repeated several times. You can do local variable `var content = self._ui.content`

Reply:
done.